flask_selfdoc/autodoc.py

```diff
@@ -201,7 +302,10 @@ def html(self, groups='all', template=None, **context):
                         raise RuntimeError(
                             "Autodoc was not initialized with the Flask app.")
 
-    def json(self, groups='all'):
+    def json(self,
+             groups='all',
+             indent: Optional[int] = None,
+             separators: Optional[Tuple] = (',', ':')):
```

Comment:
I'm adding some options that are passed to `json.dumps`, I found it practical to debug the JSON output comparisons in the unit tests. Default options produce the same output as `jsonify` (i.e. minified JSON)